eZ/Publish/SPI/FieldType/FieldType.php

```diff
@@ -190,6 +190,11 @@ abstract class FieldType
      */
     abstract public function isSingular();
 
+    public function isThumbnail(): bool
```

Comment:
this looks rather weird if you see that rest of the method is abstract, @alongosz ?

Reply:
depends on the intention, maybe it's default behavior? Though weird to see that it's `true` by default. Don't know the context here.